Dockerfile

```diff
@@ -3,6 +3,7 @@ MAINTAINER Phuong Doan pdoan@cuahsi.org
 
 # inplaceedit in pip doesn't seem compatible with Django 1.11 yet...
 RUN pip install git+https://github.com/theromis/django-inplaceedit.git@e6fa12355defedf769a5f06edc8fc079a6e982ec
+RUN pip install markdown nose-timer
```

Comment:
@cuahsimarko markdown is already moved into hs_docker_base image, so this pip install markdown line should be removed.

Reply:
@cuahsimarko it is probably better to move nose-timer into hs_docker_base image since @sblack-usu has been trying to enforce to get extra packages into the base image.

Reply:
I have opened a PR to hs_docker_base to comply https://github.com/hydroshare/hs_docker_base/pull/22

Reply:
Fixed in 8df0ec4e6

Reply:
In commit 8a3cb4488 the version of docker base image is bumped, anticipating a future 1.9.4 release after the PR in that repo is approved.